v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -0,0 +1,272 @@
+const get = require("get-value");
+const { logger } = require("handlebars");
+const { isArray } = require("lodash");
+const sha256 = require("sha256");
+const {
+  isDefinedAndNotNullAndNotEmpty,
+  returnArrayOfSubarrays,
+  CustomError,
+  constructPayload,
+  defaultRequestConfig,
+  getSuccessRespEvents,
+  getErrorRespEvents,
+  removeUndefinedAndNullAndEmptyValues
+} = require("../../util");
+const {
+  offlineDataJobsMapping,
+  addressInfoMapping,
+  BASE_ENDPOINT,
+  attributeMapping,
+  hashAttributes
+} = require("./config");
+
+const hashEncrypt = object => {
+  Object.keys(object).forEach(key => {
+    if (hashAttributes.includes(key)) {
+      // eslint-disable-next-line no-param-reassign
+      object[`${key}`] = sha256(object[`${key}`]);
+    }
+  });
+};
+
+/**
+ * This function is used for building the response. It create a default rudder response
+ * and populate headers, params and body.JSON
+ * @param {*} metadata
+ * @param {*} body
+ * @param {*} param2
+ * @returns
+ */
+const responseBuilder = (metadata, body, { Config }) => {
+  const payload = body;
+  const response = defaultRequestConfig();
+  response.endpoint = `${BASE_ENDPOINT}/${Config.customerId}/offlineUserDataJobs`;
+  response.body.JSON = removeUndefinedAndNullAndEmptyValues(payload);
+  const accessToken = metadata.secret.access_token;
+  response.params = { listId: Config.listId, customerId: Config.customerId };
+  response.headers = {
+    Authorization: `Bearer ${accessToken}`,
+    "Content-Type": "application/json",
+    "developer-token": get(metadata, "secret.developer_token")
```

Comment:
suggestion: we have wrapper functions on top of get-value. we can use those also like getValueFromMessage

Reply:
Done